scripts/src/blockchain/errors.d.ts

```diff
@@ -0,0 +1,62 @@
+export declare const HorizonErrorList: {
```

Comment:
this one was checked in by mistake (probably in tests there is an import of .ts)